Core/Core/Syllabus/SyllabusTabViewController.swift

```diff
@@ -54,6 +54,7 @@ open class SyllabusTabViewController: HorizontalMenuViewController, ColoredNavVi
         settings.refresh()
         colors.refresh()
         course.refresh()
+        trackScreenTime(eventName: "coursers/\(courseID)/assignments/syllabus")
```

Comment:
Typo: `coursers`. Also I think this should just be `courses/id/syllabus`, we go through no assignments while reaching this page.

Reply:
Fixex the typo. 

I copied this path from the web and I believe Android is doing the same.